src/main/java/ca/corefacility/bioinformatics/irida/processing/impl/FastqcFileProcessor.java

```diff
@@ -52,6 +54,7 @@
 	private final SequenceFileRepository sequenceFileRepository;
 	private final AnalysisOutputFileRepository outputFileRepository;
 	private final MessageSource messageSource;
+	private IridaFileStorageUtility iridaFileStorageUtility;
```

Comment:
This isn't being used anymore.

Reply:
This is now required as we moved the getTemporaryFile method out of the sequenceFile